app/views/task/_form_edit_task.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for @task, url: "/task/edit_update/#{@task[:id]}" do |t| %>
```

Comment:
I think you could reduce duplicate code across this view and the new view by using variables to store the different URLs and the text associated with the submit button.